File: fuzz/corpus/fuzz_scala_parser/seed_simple.scala
package com.acme.app

import scala.collection.mutable

trait Greeter {
  def greet(name: String): String
}

class Hello(prefix: String) extends Greeter {
  def greet(name: String): String = s"$prefix $name"
}

object Main {
  def run(): Unit = println(new Hello("hello").greet("world"))
}

File: fuzz/corpus/fuzz_php_parser/seed_simple.php
<?php

namespace Acme\App;

interface Greeter
{
    public function greet(string $name): string;
}

class Hello implements Greeter
{
    public function __construct(private string $prefix)
    {
    }

    public function greet(string $name): string
    {
        return $this->prefix . ' ' . $name;
    }
}
